proxy/src/lib.rs

```diff
@@ -174,7 +178,7 @@ where
             outbound_listener,
             metrics_listener,
             get_original_dst,
-            reactor: mut core,
+            runtime: mut rt,
```

Comment:
I'd prefer that `rt` be given a more descriptive name.  When I see a short name like `rt`, I assume that this is a temp variable that is going to be used immediately.  In this case, afaict, `rt` isn't used for ~150 lines.  I think it would be more appropriate to call this `runtime` or `main_runtime` or something so that it's clearer when/where it's used.

Reply:
Sounds good! I originally called it `runtime` but the tokio examples used the name `rt`, so I thought that was convention.

Reply:
Changed in c6798e9d7bb3e255c77d7fe3fdac8f0d1841b46d